pandas/core/window.py

```diff
@@ -312,6 +312,9 @@ class Window(_Window):
     * ``gaussian`` (needs std)
     * ``general_gaussian`` (needs power, width)
     * ``slepian`` (needs width).
+
+    To know more about the frequency strings (offset aliases), please visit this
+    link - http://pandas.pydata.org/pandas-docs/stable/timeseries.html#offset-aliases.
```

Comment:
see how links are done, look at ``core/frame.py/to_latex``.

To learn more about .....